app/models/form/sales/pages/buyer2_live_in_property.rb

```diff
@@ -4,12 +4,24 @@ def initialize(id, hsh, subsection)
     @id = "buyer_2_live_in_property"
     @depends_on = [
       {
-        "joint_purchase?" => true,
         "privacynotice" => 1,
+        "outright_sale?" => false,
```

Comment:
is making this conditional on not-outright sale relevant?
don't we just care about joint purchase?

Reply:
if it is not an outright sale, it must be a joint purchase (and buylivein will be nil because routing)

if it is an outright sale, buyers_will_live_in must be true (buylivein) and joint purchase true for this page to show

Reply:
coolio